generator/src/test/java/com/scottlogic/deg/generator/cucumber/Violation.feature

```diff
@@ -0,0 +1,136 @@
+Feature: User can set the generation strategy to violate mode and see test cases generated
+
+Background:
+  Given the generation strategy is interesting
+  And the combination strategy is exhaustive
+  And there is a field foo
+  And the data requested is violating
+  And the generator can generate at most 20 rows
+
+  Scenario: Running the generator in violate mode for negated equal to is successful
+   Given there is a constraint:
+       """
```

Comment:
Can we stick to using the 'cucumber' expressions for these constraints, it would help - i think - to clarify the test.

e.g.
`Given foo is anything but equal to 8`